src/hardware/memory.cpp

```diff
@@ -426,6 +426,14 @@ bool MEM_A20_Enabled(void) {
 	return memory.a20.enabled;
 }
 
+// Use this function for initialization since the public 
+// function is optimized to return on the same state 
+static void A20_Enable_Force(bool enabled) {
```

Comment:
"Enable Force enabled": the name is pretty boolean-heavy.

The `CAPS_<function>` is the pattern for exported functions (MEM_A20_Enable is public).

But this functions in just static/local, so we can use basic CamelCase.

Maybe `"InitializeA20(const bool enabled)"`? Or something to that effect?

I'm not sure if it's an initializer or if we expect it to be called repeatedly?

Reply:
Hmm.. if the only use-case is to disable the A20 memory, maybe the bool-arg isn't even needed, `InitializeA20()`?

Reply:
> Hmm.. if the only use-case is to disable the A20 memory, maybe the bool-arg isn't even needed, `InitializeA20()`?

Hah! This was my original plan. I thought I'd add the boolean just in case it ever needed to be called to enable it.

But very happy to do the InitializeA20. Incoming!